test/metabase/query_processor/middleware/pre_alias_aggregations_test.clj

```diff
@@ -74,32 +74,28 @@
 
 (deftest ^:parallel expressions-test
   (testing "does pre-aliasing work the way we'd expect with expressions?"
-    (is (= {:database 1
-            :type     :query
-            :query    {:source-table 1
-                       :aggregation  [[:aggregation-options [:+ 20 [:sum [:field 2 nil]]] {:name "expression"}]]}}
+    (is (= (mt/mbql-query checkins
+             {:source-table $$venues
+              :aggregation  [[:aggregation-options [:+ 20 [:sum $user_id]] {:name "expression"}]]})
            (pre-alias
-            {:database 1
-             :type     :query
-             :query    {:source-table 1
-                        :aggregation  [[:+ 20 [:sum [:field 2 nil]]]]}})))))
+            (mt/mbql-query checkins
+              {:source-table $$venues
+               :aggregation  [[:+ 20 [:sum $user_id]]]}))))))
 
 (deftest ^:parallel expressions-test-2
-  (is (= {:database 1
-          :type     :query
-          :query    {:source-table 1
-                     :aggregation  [[:aggregation-options
-                                     [:+ 20 [:sum [:field 2 nil]]]
-                                     {:name "expression"}]
-                                    [:aggregation-options
-                                     [:- 20 [:sum [:field 2 nil]]]
-                                     {:name "expression_2"}]]}}
+  (is (= (mt/mbql-query checkins
+           {:source-table $$venues
+            :aggregation  [[:aggregation-options
+                            [:+ 20 [:sum $user_id]]
+                            {:name "expression"}]
+                           [:aggregation-options
+                            [:- 20 [:sum $user_id]]
+                            {:name "expression_2"}]]})
          (pre-alias
-          {:database 1
-           :type     :query
-           :query    {:source-table 1
-                      :aggregation  [[:+ 20 [:sum [:field 2 nil]]]
-                                     [:- 20 [:sum [:field 2 nil]]]]}}))))
+          (mt/mbql-query checkins
+            {:source-table $$venues
+             :aggregation  [[:+ 20 [:sum $user_id]]
+                            [:- 20 [:sum $user_id]]]})))))
```

Comment:
These changes fix some nonsense aggregations, they are not strictly necessary for this PR.